packages/clerk-js/src/ui/components/SignUp/SignUpPhoneCodeCard.tsx

```diff
@@ -1,19 +1,21 @@
-import React from 'react';
+import { useEffect } from 'react';
 
 import { useCoreSignUp } from '../../contexts';
 import { Flow, localizationKeys } from '../../customizables';
 import { withCardStateProvider } from '../../elements';
+import { useEffectForceOnce } from '../../hooks/useEffectForceOnce';
 import { SignUpVerificationCodeForm } from './SignUpVerificationCodeForm';
 
 export const SignUpPhoneCodeCard = withCardStateProvider(() => {
   const signUp = useCoreSignUp();
 
-  React.useEffect(() => {
-    // TODO: This prepare method is not idempotent.
-    // We need to make sure that R18 won't trigger this twice
-    void prepare();
+  useEffectForceOnce(() => console.log('NICE'));
+  useEffect(() => {
+    console.log('nice');
```

Comment:
Nice :)

Reply:
I'm debugging on another app that is not local in my machine and i need the logs + snapshots :)